### ISSUE TITLE
Orphaned form label within Editorial Workflow component of the CMS

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#10113]
## Description
**Still an issue in D10**

In the Editorial workflow component on the edit screens in the CMS, the "Current State" is displayed using the `<label>` element.  However, there is no associated field causing this to be an orphaned form label.

## Screenshot
![image](https://user-images.githubusercontent.com/106678594/183679258-378ce3f9-9292-4bdb-b4fd-480967fdfdd7.png)

## Additional Content
There is a [different ticket](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/10122) for orphaned form labels in Facilities products on the Read only sections on content.

That ticket also mentioned that I found some D.o issues that might be related to this issue but I'm not sure if they are exactly the same:
- https://www.drupal.org/project/drupal/issues/884846
- https://www.drupal.org/project/drupal/issues/2279111

## Accessibility Standard

WCAG version 2.0 A, [Criterion 1.3.1](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html)

## Acceptance Criteria
- [ ] Determine if this is a core issue
- [ ] Based on above, determine path forward for remediation (i.e. can it be completed within this ticket, do we need to create a new ticket, etc.)
- [ ] Accessibility Review

## Previous Team Points
5

**Comment by laflannery** [comment]
This might be hard to update globally.

It might be possible to do some sort of conditional like if no children set as span? How would we keep if from called the label from the other template?

**Comment by ndouglas** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Accessibility-Issues-62b4be010177900017e82254/issues/department-of-veterans-affairs/va.gov-cms/10113?planning-poker) with Zenhub @edmund-dunn @tonytaylor

**Comment by srancour** [comment]
This is still an issue that should be addressed.

**Comment by anantais** [comment]
This fix went to github today. Waiting on Steve to QA.
